package.json

```diff
@@ -95,7 +95,8 @@
     "prettier": "^1.6.1",
     "shx": "^0.2.2",
     "stylelint": "^8.1.1",
-    "stylelint-config-standard": "^17.0.0"
+    "stylelint-config-standard": "^17.0.0",
+    "is-reachable":"^2.3.3"
```

Comment:
can you move this to below `"eslint-plugin-prettier"` so that it's alphabetical and doesn't move somewhere else when we do `npm install --save`.